source/docs/software/driverstation/manually-setting-the-driver-station-to-start-custom-dashboard.rst

```diff
@@ -31,9 +31,9 @@ DashboardCmdLine
 
 Locate the line beginning with ``DashboardCmdLine``.
 
-Replace the string after ``=`` with ``java -jar "C:\\PATH\\TO\\DASHBOARD.jar"`` where the path specified is the path to the dashboard ``jar`` file. Save the ``FRC DS Data Storage`` file.
+Replace the string after ``=`` with ``Java -jar "C:\\PATH\\TO\\DASHBOARD.jar"`` where the path specified is the path to the dashboard ``jar`` file. Save the ``FRC DS Data Storage`` file.
```

Comment:
Shouldn't change java to Java in command line invocations in case of case sensitive file systems.